Microsoft.Dynamics365.UIAutomation.Api.UCI/DTO/ElementReference.cs

```diff
@@ -184,7 +184,7 @@ public static class Elements
             { "Login_SignIn", "id(\"cred_sign_in_button\")"},
             { "Login_CrmMainPage", "//*[contains(@id,'crmTopBar') or contains(@data-id,'topBar')]"},
             { "Login_CrmUCIMainPage", "//*[contains(@data-id,'topBar')]"},
-            { "Login_StaySignedIn", "//input[@id=\"idSIButton9\"]"},
+            { "Login_StaySignedIn", "//div[@data-viewid and contains(@data-bind, 'kmsi-view')]//input[@id='idSIButton9']"},
```

Comment:
Click on KMSI button, in some scenario another Dialog / Button was shown & clicked by the logic.